_includes/v22.1/backups/backup-options.md

```diff
@@ -2,6 +2,6 @@
 -----------------------------------------------------------------+-------------------------+------------------------------
 `revision_history`<a name="with-revision-history"></a>           | N/A                     | Create a backup with full [revision history](take-backups-with-revision-history-and-restore-from-a-point-in-time.html), which records every change made to the cluster within the garbage collection period leading up to and including the given timestamp.
 `encryption_passphrase`<a name="with-encryption-passphrase"></a> | [`STRING`](string.html) | The passphrase used to [encrypt the files](take-and-restore-encrypted-backups.html) (`BACKUP` manifest and data files) that the `BACKUP` statement generates. This same passphrase is needed to decrypt the file when it is used to [restore](take-and-restore-encrypted-backups.html) and to list the contents of the backup when using [`SHOW BACKUP`](show-backup.html). There is no practical limit on the length of the passphrase.
-`DETACHED`<a name="detached"></a>                           | N/A                     |  When a backup runs in `DETACHED` mode, it will execute asynchronously and the job ID will be returned immediately without waiting for the job to finish. Note that with `DETACHED` specified, further job information and the job completion status will not be returned. For more on the differences between the returned job data, see the [example](backup.html#run-a-backup-asynchronously) below. To check on the job status, use the [`SHOW JOBS`](show-jobs.html) statement. <br><br>To run a backup within a [transaction](transactions.html), use the `DETACHED` option.
+`DETACHED`<a name="detached"></a>                           | N/A                     |  When a backup runs in `DETACHED` mode, it will execute asynchronously. The job ID will be returned after the backup is validated. Note that with `DETACHED` specified, further job information and the job completion status will not be returned. For more on the differences between the returned job data, see the [example](backup.html#run-a-backup-asynchronously) below. To check on the job status, use the [`SHOW JOBS`](show-jobs.html) statement. <br><br>To run a backup within a [transaction](transactions.html), use the `DETACHED` option.
```

Comment:
For v22.1 and v22.2, we have a backup architecture page. One section actually talks about the how the job is created. It might be worth linking [backup is validated](backup-architecture.html#job-creation-phase) in your new description to this section ^.